Sources/Purchasing/Purchases.swift

```diff
@@ -523,6 +523,18 @@ extension Purchases {
         subscriberAttributesManager.setPushToken(pushToken, appUserID: appUserID)
     }
 
+    /**
+     * Subscriber attribute associated with the push token for the user.
+     *
+     * #### Related Articles
+     * -  [Subscriber attributes](https://docs.revenuecat.com/docs/subscriber-attributes)
+     *
+     * - Parameter pushToken: `nil` will delete the subscriber attribute.
+     */
+    @objc public func setPushTokenString(_ pushToken: String?) {
```

Comment:
What do you think about overloading the name in Swift, and only changing it in Objective-C since it doesn't support it?
```suggestion
    @objc(setPushTokenString:)
    public func setPushToken(_ pushToken: String?) {
```

Reply:
I should have added a comment on the PR to clarify that in advance

Reply:
Did you try `Purchases/setPushToken(_: String)` and `Purchases/setPushToken(_: Data)`?
I wonder if the "better" name is worth more than linking both methods with a DocC-string.

Reply:
<img width="824" alt="image" src="https://user-images.githubusercontent.com/3922667/165134910-3c3bf746-a6d3-422e-bc2c-e9c8843e856a.png">

I'd love to get that working, though, I just don't know a good way to do it

Reply:
We could just not add the related symbol? 🤷🏻‍♂️

Reply:
sure, but the other place where it happens is in Purchases.md([renders this site](https://revenuecat.github.io/purchases-ios-docs/4.2.1/documentation/revenuecat/purchases)) and in RevenueCat.md ([renders this site](https://revenuecat.github.io/purchases-ios-docs/4.2.1/documentation/revenuecat/)), which serve as indexes for public methods. 

If we add most methods in there but leave out any, they won't be rendered in the Revenuecat page at all, and they'll only show up at the bottom of the Purchases.md site

Reply:
Not worth wasting more time then, ship it!